frontend/src/pages/LogsPage/SearchForm/utils.ts

```diff
@@ -84,3 +89,35 @@ export const stringifyLogsQuery = (params: LogsSearchParam[]) => {
 export const validateLogsQuery = (params: LogsSearchParam[]): boolean => {
 	return !params.some((param) => !param.value)
 }
+
+export const getLogsURLForSession = (projectId: string, session: Session) => {
+	const queryParams: LogsSearchParam[] = []
+	queryParams.push({
+		key: ReservedLogKey.SecureSessionId,
+		operator: DEFAULT_LOGS_OPERATOR,
+		value: session.secure_id,
+		offsetStart: 0,
+	})
+
+	const sessionStartDate = new Date(session.created_at)
+
+	// Instead of doing `new Date()` (equivalent to the time now), we set an arbitrary end date of one day after the session was created.
+	// We have found that Clickhouse doesn't perform well with long time ranges (see https://github.com/highlight/highlight/pull/4652)
+	// Rarely will a session last longer than a day. If users need to handle this case, we should figure out a way for the backend to optimize
+	// long time ranges.
+	const sessionEndDate = moment(sessionStartDate).add(1, 'day').toDate()
```

Comment:
could even set this to 4 hours which is the limit to session length upon processing and in the browser client

Reply:
Nice tip. Fixed in 981b8ae728e7e11879cd86b7db79d34fcec384da